src/openloco/industry.cpp

```diff
@@ -111,13 +111,13 @@ namespace openloco
         }
     }
 
-    void industry::sub_454A43(coord_t x, coord_t y, uint8_t bl, uint8_t bh, uint8_t dl)
+    void industry::sub_454A43(coord_t ax, coord_t cx, uint8_t bl, uint8_t bh, uint8_t dl)
```

Comment:
ax and cx are still an x and y but they represent a random tile near the industry item. I would like to see this replaced with a `map::map_pos` as it is a reference to a tile.

Reply:
like this?
```
void industry::sub_454A43(map_pos pos, uint8_t bl, uint8_t bh, uint8_t dl)
{
    registers regs;
    regs.bl = bl;
    regs.bh = bh;
    regs.ax = pos.x;
    regs.cx = pos.y;
    regs.dl = dl;
    regs.dh = id();
    call(0x00454A43, regs);
}
```

Reply:
Yup that looks better